package.json

```diff
@@ -61,7 +61,7 @@
     "@ethersproject/units": "5.7.0",
     "@ethersproject/wallet": "5.7.0",
     "@openzeppelin/contracts": "4.6.0",
-    "@rari-capital/solmate": "6.4.0",
+    "solmate": "https://github.com/transmissions11/solmate",
```

Comment:
we should fix the dependency adding `#thecommithashwewant` at the end of the git url, so t11s pushing to master doesn't inadvertedly change what code we get in the dependency